src/Project/ToolRunner.ts

```diff
@@ -57,6 +87,35 @@ export class ToolRunner {
     });
   }
 
+  /**
+   * Function to kill child process
+   */
+  public kill(): boolean {
+    this.killedByMe = false;
+
+    if (this.child === undefined || this.child.killed) {
+      throw Error('No process to kill');
+    }
+
+    const signals: NodeJS.Signals[] = ['SIGINT', 'SIGTERM', 'SIGKILL'];
```

Comment:
AFAIK, `SIGTERM` is for exiting program as normal as possible.
But pressing `Cancel` button do not seem to be normal I think.
Could you let me know why `SIGTERM` is included here? :)

Reply:
I referred to this article: https://www.baeldung.com/linux/sigint-and-other-termination-signals#how-sigint-relates-to-sigterm-sigquit-and-sigkill

I didn't try SIGQUIT since it generates core dump.